maker/src/cfd.rs

```diff
@@ -47,6 +50,16 @@ use xtras::SendAsyncSafe;
 const HANDLE_ACCEPT_CONTRACT_SETUP_MESSAGE_TIMEOUT: std::time::Duration =
     std::time::Duration::from_secs(10);
 
+/// Determine which maia version to use in protocol execution
+///
+/// Older version of maia had a bug in which the CETs were calculated incorrectly.
+/// This enum ensures that we can be backwards-compatible with takers running older version.
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum MaiaProtocolVariant {
+    Old,
```

Comment:
🙃 How about one of these instead:

* `V1` and `V2`
* The old maia version that contains the bug as `x.y.z` and `CURRENT`